tasties_app/templates/tasties_app/base.html

```diff
@@ -82,7 +82,17 @@
     </nav>
 
     <!-- Main content -->
-    {% block content %}{% endblock %}
+    {% block content %}
```

Comment:
Please give the block a different name, because there is already a block named `content` and it is used by other templates that inherit this one.